qucs/autogen.sh

```diff
@@ -31,6 +31,8 @@ echo "done."
 echo -n "Creating config.h.in... "
 autoheader
 echo "done."
+echo "Running libtoolize"
+libtoolize || exit 1
```

Comment:
I did not read any of the code, but this should fix the OS X  build:
https://github.com/Qucs/qucs/blob/master/qucs-core/bootstrap.sh#L53-L58